packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -526,13 +526,13 @@ class _DraggableSheetExtent {
   void updateSize(double newSize, BuildContext context) {
     assert(newSize != null);
     _currentSize.value = newSize.clamp(minSize, maxSize);
-    DraggableScrollableNotification(
+    context.dispatchNotification(DraggableScrollableNotification(
```

Comment:
revert this change since calling `dispatch` continues to be valid?

Reply:
done